src/native/overlay/list_menu.rs

```diff
@@ -1,417 +0,0 @@
-                    );
-                }
-            }
-            Event::Touch(touch::Event::FingerPressed { .. }) => {
-                let bounds = layout.bounds();
-
-                if bounds.contains(cursor_position) {
-                    let text_size = self.text_size.unwrap_or_else(|| renderer.default_size());
-
-                    *self.hovered_option = Some(
-                        ((cursor_position.y - bounds.y) / f32::from(text_size + (self.padding * 2)))
-                            as usize,
-                    );
-
-                    if let Some(index) = *self.hovered_option {
-                        if let Some(option) = self.options.get(index) {
-                            *self.last_selection = Some(option.clone());
-                        }
-                    }
-                }
-            }
-            _ => {}
-        }
-
-        event::Status::Ignored
-    }
-
-    fn draw(
-        &self,
-        renderer: &mut Renderer,
-        _defaults: &Renderer::Defaults,
-        layout: Layout<'_>,
-        cursor_position: Point,
-        viewport: &Rectangle,
-    ) -> Renderer::Output {
-        self::Renderer::draw(
-            renderer,
-            layout.bounds(),
-            cursor_position,
-            viewport,
-            self.options,
-            *self.hovered_option,
-            self.padding,
-            self.text_size.unwrap_or_else(|| renderer.default_size()),
-            self.font,
-            &self.style,
-        )
-    }
-}
-
-/// The renderer of a [`ListMenu`].
-///
-/// Your [renderer] will need to implement this trait before being
-/// able to use a [`ListMenu`] in your user interface.
-///
-/// [renderer]: crate::renderer
-pub trait Renderer: scrollable::Renderer + container::Renderer + text::Renderer {
-    /// The [`ListMenu`] style supported by this renderer.
-    type Style: Default + Clone;
-
-    /// Decorates a the list of options of a [`ListMenu`].
-    ///
-    /// This method can be used to draw a background for the [`ListMenu`].
-    fn decorate(
-        &mut self,
-        bounds: Rectangle,
-        cursor_position: Point,
-        style: &<Self as Renderer>::Style,
-        primitive: Self::Output,
-    ) -> Self::Output;
-
-    /// Draws the list of options of a [`ListMenu`].
-    #[allow(clippy::too_many_arguments)]
-    fn draw<T: ToString>(
-        &mut self,
-        bounds: Rectangle,
-        cursor_position: Point,
-        viewport: &Rectangle,
-        options: &[T],
-        hovered_option: Option<usize>,
-        padding: u16,
-        text_size: u16,
-        font: Self::Font,
-        style: &<Self as Renderer>::Style,
-    ) -> Self::Output;
-}
-
-impl<'a, T, Message, Renderer> From<List<'a, T, Renderer>> for Element<'a, Message, Renderer>
-where
-    T: ToString + Clone,
-    Message: 'a,
-    Renderer: 'a + self::Renderer,
-{
-    fn from(list: List<'a, T, Renderer>) -> Element<'a, Message, Renderer> {
-        Element::new(list)
-    }
-}
```

Comment:
This file could now be placed in a submodule of the selection list instead of the overlay module